openbb_platform/CONTRIBUTING.md

```diff
@@ -451,66 +533,38 @@ Please refer to [OpenBBTerminal docs](https://docs.openbb.co/terminal/installati
-<summary>Install all dependencies in editable mode at once</summary>
+   > To install all the packages, including extras, use the `-e` argument with the above script.
 
-For development purposes, one can install every available extension by running a custom shell script:
+## How to create a PR?
 
-Navigate to `/OpenBBTerminal/openbb_platform`
+To create a PR to the OpenBB Platform, you'll need to fork the repository and create a new branch.
 
-Run `python dev_install.py`.
-</details>
+1. Create your Feature Branch, e.g. `git checkout -b feature/AmazingFeature`
+2. Check the files you have touched using `git status`
+3. Stage the files you want to commit, e.g.
+   `git add openbb_terminal/stocks/stocks_controller.py openbb_terminal/stocks/stocks_helper.py`.
+   Note: **DON'T** add any files with personal information.
+4. Write a concise commit message under 50 characters, e.g. `git commit -m "meaningful commit message"`. If your PR
+   solves an issue raised by a user, you may specify such issue by adding #ISSUE_NUMBER to the commit message, so that
+   these get linked. Note: If you installed pre-commit hooks and one of the formatters re-formats your code, you'll need
+   to go back to step 3 to add these.
 
-> In order to install any other custom extension or provider, you'd follow the exact same steps as above.
+### Install pre-commit hooks
 
-## Python package
+To install pre-commit hooks, run `pre-commit install` in the root of the repository.
 
-### Overview
+### Branch Naming Conventions
 
-One of the interfaces we provide to users is the `openbb` python package.
+The accepted branch naming conventions are:
 
-The code you will find in this package is generated from a script and it is just a
-wrapper around the `openbb-core` and any installed extensions.
+- `feature/feature-name`
+- `hotfix/hotfix-name`
+- `release/2.1.0` or `release/2.1.0rc0`.
 
-When the user runs `import openbb`, `from openbb import obb` or other variants, the
-script that generates the package code is triggered. It detects if there are new openbb
-extensions installed in the environment and rebuilds the package code accordingly. If
-new extensions are not found it just uses the current package version.
-
-When you are developing chances are you want to manually trigger the package rebuild.
-You can do that with:
-
-```python
-python -c "import openbb; openbb.build()"
-```
-
-### Import time
-
-We aim to have a short import time for the package. To measure that we use `tuna`.
-
-- https://pypi.org/project/tuna/
-
-To visualize the import time breakdown by module and find potential bottlenecks, run the
-following commands from `openbb_platform` directory:
-
-```bash
-pip install tuna
-python -X importtime openbb/__init__.py 2> import.log
-tuna import.log
-```
+All `feature/feature-name` related branches can only have PRs pointing to `develop` branch. `hotfix/hotfix-name` and `release/2.1.0` or `release/2.1.0rc0` branches can only have PRs pointing to `main` branch.
```

Comment:
I am under the impression that nobody merges to `main`, and that `/hotfix/patch-name` should be a branch created from `/develop`.

Reply:
I copied this part from the v3 CONTRIBUTING.md docs. We can remove it.

Reply:
Resolved! Thanks for pointing this @deeleeramone